{"name": "test_login_negative_shows_error[standard_user-]", "status": "passed", "steps": [{"name": "Логин", "status": "passed", "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "''"}], "start": 1757407194685, "stop": 1757407195087}], "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "''"}], "start": 1757407194607, "stop": 1757407195121, "uuid": "62aea299-4312-4208-a380-7ada497de8b4", "historyId": "b099ffad2fe812c92aeff674440c9674", "testCaseId": "4075f5f56554ec24a79f4d66f84a44d9", "fullName": "tests.test_auth#test_login_negative_shows_error", "labels": [{"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "Zak"}, {"name": "thread", "value": "12376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}], "titlePath": ["tests", "test_auth.py"]}